crates/kitsune_p2p/types/src/bin_types.rs

```diff
@@ -160,6 +160,52 @@ impl KitsuneOpData {
     }
 }
 
+/// Helpful pattern for debug formatting many bytes.
+/// If the size is > 32 bytes, only the first 10 and last 10 bytes will be displayed.
```

Comment:
its actually doing first and last 8... also, isn't this duplicated elsewhere in this PR?

Reply:
It's duplicated once in kitsune and once in holochain_integrity_types. No common crate between them :/